exodus/trackers/templates/trackers_list.html

```diff
@@ -7,15 +7,11 @@
       <h1 class="main-title">
         {% trans "Trackers" %}
       </h1>
-      <h4>
+      <h5>
```

Comment:
Why this change?
I would have kept h4 for both pages subtitle (Trackers & Reports)

Reply:
Alright, found it a bit heavy and did not want to check CSS =)
But brought back h4